src/pages/SwapPage/CommunicationCard.tsx

```diff
@@ -0,0 +1,60 @@
+import { faHandshake } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { CardHeader } from "@material-ui/core";
+import SendIcon from "@material-ui/icons/Send";
+import React from "react";
+import { CommunicationStatus } from "../../api/swapResource";
+import { Role } from "../../api/swapTypes";
+
+interface CommunicationCardHeaderProps {
+  status: CommunicationStatus;
+  role: Role;
+}
+
+function CommunicationCardHeader({
+  status,
+  role
+}: CommunicationCardHeaderProps) {
+  /* TODO: Display swap decline reason if available */
+  /* TODO: Add retry button */
```

Comment:
Some serious scope creep here? I am not aware that we have the ability to retry the sending :D

Reply:
Hahah, I did not mean to imply that it all should be done with this PR! I can delete the TODOs if you prefer.